### DISCUSSION TITLE
Feature requests

### DISCUSSION THREAD

**Discussion by sadden3194** [mudkipme/MoeMemos#123] [Ideas]
Hi @mudkipme,

I have some idea to make this app better: 

- using this app offline with a later sync process, if internet is available again

- another app icon - like a notes symbol. Maybe the option to select an individual icon from a list of app icons

- WYSIWYG editor
or if WYSIWYG editor is non a option:

- more "markdown buttons":
-- headlines
-- Bold 
-- Italic 
-- Underline
-- Table

**Comment by mudkipme** [comment]
Thanks for your suggestion!

I do plan to implement new features such as offline support, more app icons and markdown toolbar in major version upgrade.
